README.md

```diff
@@ -6,9 +6,132 @@
 ![Docker](https://img.shields.io/badge/Docker-%232496ED.svg?logo=docker&logoColor=white&style=flat-square)
 [![CI status](https://img.shields.io/github/actions/workflow/status/songstitch/anchor/ci.yaml?branch=main&style=flat-square&logo=github)](https://github.com/SongStitch/anchor/actions?query=branch%3Amain)
 [![License](https://img.shields.io/github/license/SongStitch/anchor?style=flat-square)](/LICENSE)
+[![Release](https://img.shields.io/github/v/release/SongStitch/anchor?style=flat-square)](https://github.com/SongStitch/anchor/releases/latest)
 
-</div>
-
-<div align="center">
 A tool for anchoring dependencies in dockerfiles
+
 </div>
+
+<!-- toc -->
+
+- [Installation](#installation)
+  - [Via Go Install](#via-go-install)
+  - [Via GitHub Releases](#via-github-releases)
+- [What is Anchor, and How Does it Work?](#what-is-anchor-and-how-does-it-work)
+  - [By Example](#by-example)
+- [Supported Operating Systems Package Managers](#supported-operating-systems-package-managers)
+- [Recommended Workflow](#recommended-workflow)
+- [Usage](#usage)
+  - [Default Usage](#default-usage)
+  - [Specifying Input and Output Files](#specifying-input-and-output-files)
+  - [Non-Interactive Mode (CI/CD Pipelines)](#non-interactive-mode-cicd-pipelines)
+  - [Printing the Output Instead of Writing to a File](#printing-the-output-instead-of-writing-to-a-file)
+- [License](#license)
+
+<!-- tocstop -->
+
+# Installation
+
+## Via Go Install
+
+```shell
+go install github.com/songstitch/anchor@latest
+```
+
+## Via GitHub Releases
+
+Download the latest binary from the [releases page](https://github.com/SongStitch/anchor/releases/latest)
+
+# What is Anchor, and How Does it Work?
+
+Anchor is a tool for anchoring (or pinning) Dockerfiles. It allows for reproducible builds by ensuring that the versions of dependencies are fixed. This is done in two ways
+
+- Replacing docker image tags referenced in a Dockerfile with the digest of the image
+- Replacing package versions in a Dockerfile with the version of the package. The parent digest image is used resolve the package versions to ensure that the package versions are consistent with the parent image.
+
+Anchor is designed to be a drop-in tool that can be used against existing Dockerfiles without the need to modify them.
```

Comment:
I wonder if it's worthwhile making the comparison with lock files used for package managers? And maybe also highlighting the fact that this is just used to generate an anchored dockerfile, meaning there's no changes that are needed as part of your CI or whatever to build the image.